ruleset/rules/0755-office365_rules.xml

```diff
@@ -0,0 +1,928 @@
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^18$</field>
+        <description>Office 365: Admin actions from the Security and Compliance Center.</description>
+        <options>no_full_log</options>
+        <group>SecurityComplianceCenterEOPCmdlet</group>
+    </rule>
+
+    <rule id="91549" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^19$</field>
+        <description>Office 365: Aggregated Exchange mailbox auditing events.</description>
+        <options>no_full_log</options>
+        <group>ExchangeAggregatedOperation</group>
+    </rule>
+
+    <rule id="91550" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^20$</field>
+        <description>Office 365: Power BI events.</description>
+        <options>no_full_log</options>
+        <group>PowerBIAudit</group>
+    </rule>
+
+    <rule id="91551" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^21$</field>
+        <description>Office 365: Dynamics 365 events.</description>
+        <options>no_full_log</options>
+        <group>CRM</group>
+    </rule>
+
+    <rule id="91552" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^22$</field>
+        <description>Office 365: Yammer events.</description>
+        <options>no_full_log</options>
+        <group>Yammer</group>
+    </rule>
+
+    <rule id="91553" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^23$</field>
+        <description>Office 365: Skype for Business events.</description>
+        <options>no_full_log</options>
+        <group>SkypeForBusinessCmdlets</group>
+    </rule>
+
+    <rule id="91554" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^24$</field>
+        <description>Office 365: Events for eDiscovery activities performed by running content searches and managing eDiscovery cases in the Security and Compliance Center.</description>
+        <options>no_full_log</options>
+        <group>Discovery</group>
+    </rule>
+
+    <rule id="91555" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^25$</field>
+        <description>Office 365: Events from Microsoft Teams.</description>
+        <options>no_full_log</options>
+        <group>MicrosoftTeams</group>
+    </rule>
+
+    <rule id="91556" level="12">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^28$</field>
+        <description>Office 365: Phishing and malware events from Exchange Online Protection and Microsoft Defender for Office 365.</description>
+        <options>no_full_log</options>
+        <group>ThreatIntelligence</group>
+    </rule>
+
+    <rule id="91557" level="3">
```

Comment:
I think this level should be higher since these alerts come from Microsoft Defender.

Reply:
Done